hoot-core-test/src/test/cpp/hoot/core/conflate/highway/HighwaySnapMergerTest.cpp

```diff
@@ -304,7 +304,7 @@ class HighwaySnapMergerTest : public HootTestFixture
 
     TestUtils::mkpath("tmp");
     MapProjector::projectToWgs84(map);
-    OsmMapWriterFactory::write(map, "tmp/dum.osm");
+    OsmMapWriterFactory::write(map, "tmp/HighwaySnapMergerTest.osm");
```

Comment:
This filename was used in multiple tests and could be causing some random failures.  I removed all instances of `tmp/dum.osm` and replaced it with the name of the test to reduce potential for collisions.